.github/workflows/ci_pipeline.yml

```
@@ -57,6 +57,14 @@ jobs:
             python3-pip \
             python3-vcstool
 
+      - name: Install python packages
+        run: >-
+             PIP_BREAK_SYSTEM_PACKAGES=1 pip install pre-commit==2.20.0 \
+             plyfile==1.0.3 \
+             h5py==3.11.0
```

@serraramiro1 nit: 
```suggestion
             export PIP_BREAK_SYSTEM_PACKAGES=1
             pip install \
                 pre-commit==2.20.0 \
                 plyfile==1.0.3 \
                 h5py==3.11.0
```
for readability.

---

Done